src/main/java/com/aws/greengrass/clientdevices/auth/ClientDevicesAuthService.java

```diff
@@ -198,6 +199,8 @@ protected void shutdown() throws InterruptedException {
     @Override
     public void postInject() {
         super.postInject();
+        context.put(SessionConfig.class, SessionConfig.from(getConfig()));
```

Comment:
this _needs_ to be in postInject() to be able to inject the same SessionConfig (with the subscription to config changes) in all the CDA service dependencies.